fdbcli/fdbcli.actor.cpp

```diff
@@ -989,6 +989,80 @@ void printStatus(StatusObjectReader statusObj, StatusClient::StatusLevel level,
 
 				if (statusObjConfig.get("log_routers", intVal))
 					outputString += format("\n  Desired Log Routers    - %d", intVal);
+
+				outputString += "\n  Usable Regions         - ";
+				if (statusObjConfig.get("usable_regions", intVal)) {
+					outputString += std::to_string(intVal);
+				} else {
+					outputString += "unknown";
+				}
+
+				Optional<std::string> activePrimaryDC;
+
+				if (statusObjConfig.has("active_primary_dc")) {
+					activePrimaryDC = statusObjConfig["active_primary_dc"].get_str();
+				}
+
+				StatusArray regions;
+				if (statusObjConfig.has("regions")) {
+					outputString += "\n  Regions: ";
+					regions = statusObjConfig["regions"].get_array();
+					bool isFirstRegion = true;
+					bool isPrimary = false;
+					std::vector<std::string> regionSatelliteDCs;
+					std::string regionDC;
+					for (StatusObjectReader region : regions) {
+						// Assumes the `regions` are already sorted by priority in descending order
```

Comment:
Is this a property that is enforced in the code, or is it based on the expectation that a user configures a cluster with this ordering?

Reply:
It seems like so. In `DatabaseConfiguration` it sort the regions vector by priority whenever there is an update for `regions` key in the JSON provided.